src/request/import/ImportApi.js

```diff
@@ -75,7 +75,7 @@ class ImportApi extends TopLevelApi {
             );
             handler.run();
         });
-        $createWalletLink.addEventListener('click', () => alert('Sorry, not implemented yet.'));
+        $createWalletLink.addEventListener('click', () => this.reject(new Error('GOTO_CREATE')));
```

Comment:
GOTO_CREATE seems like it should be a constant. But I guess it's only used here and in accounts?

Reply:
Sebastian is working on making all errors constants currently in his `sebastian/errors` branch.